packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -77,25 +80,29 @@ document.getElementById('leave').addEventListener('click', () => {
 
 document.getElementById('start-sending-audio').addEventListener('click', () => {
   if (meeting) {
-    meeting.startSendingAudio();
+    meeting.toggleAudioVideo({audio: true, video});
+    audio = true;
   }
 });
 
 document.getElementById('stop-sending-audio').addEventListener('click', () => {
   if (meeting) {
-    meeting.stopSendingAudio();
+    meeting.toggleAudioVideo({audio: false, video});
+    audio = false;
   }
 });
 
 document.getElementById('start-sending-video').addEventListener('click', () => {
   if (meeting) {
-    meeting.startSendingVideo();
+    meeting.toggleAudioVideo({audio, video: true});
+    video = true;
   }
 });
 
 document.getElementById('stop-sending-video').addEventListener('click', () => {
   if (meeting) {
-    meeting.stopSendingVideo();
+    meeting.toggleAudioVideo({audio, video: false});
```

Comment:
Instead of requiring users to pass both audio/video can we just have them pass whatever they want to change and keep the old state the same?